### ISSUE TITLE
Augmented Network Security Rules still not allowing lists

### ISSUE THREAD

**Issue by keeperpl** [terraform-providers/terraform-provider-azurerm#863]
### Terraform Version

Terraform v0.11.3
+ provider.azurerm v1.1.2

### Affected Resource(s)

azurerm_network_security_rule

### Terraform Configuration Files
```hcl
resource "azurerm_network_security_rule" "xxxxx" {
  name                        = "xxxxx"
  description                 = "xxxxx"
  network_security_group_name = "xxxxx"
  direction                   = "Inbound"
  access                      = "Allow"
  priority                    = "150"
  source_address_prefix       = "xx.xx.xx.xx/xx"
  source_port_range           = "*"
  destination_address_prefix  = "*"
  destination_port_range      = ["161","155","88-90"]
  protocol                    = "tcp"
  resource_group_name         = "${azurerm_resource_group.resource_group.name}"
}
```

### Expected Behavior

An augmented network security group should be created as - if i understand correctly - the issue was already fixed with proper pull request.

### Actual Behavior
terraform plan

Error: 
```hcl
azurerm_network_security_rule.xxxxx: destination_port_range must be a single value, not a list
```
### Steps to Reproduce

plan/apply a rule with list of ports/addresses.

### References
https://github.com/terraform-providers/terraform-provider-azurerm/issues/659

https://github.com/terraform-providers/terraform-provider-azurerm/pull/692

**Comment by tombuildsstuff** [comment]
hi @keeperpl 

Thanks for opening this issue :)

Support for specifying the Destination Port Ranges as a List was added in #692 which was released in v1.1.2 of the Azure Provider (yesterday) - this is exposed as [the `destination_port_ranges` field](https://www.terraform.io/docs/providers/azurerm/r/network_security_rule.html#destination_port_ranges) - which is distinct [from the `destination_port_range ` field](https://www.terraform.io/docs/providers/azurerm/r/network_security_rule.html#destination_port_range) since they're different data types (a List vs a String). Would you be able to take a look and see if that solves your issue?

Thanks!

**Comment by keeperpl** [comment]
Yes, it's all working if used with "destination_port_ranges" :)
Thank you!
